.azure-pipelines/default.yml

```diff
@@ -0,0 +1,28 @@
+trigger:
+- master
+
+jobs:
+- job: 'Windows'
+
+  pool:
+    vmImage: 'vs2017-win2016'  
+
+  steps:
+  - script: |
+     python -m venv .\
+     
+     pip install git+https://github.com/lmazuel/swagger-to-sdk
+     pip install -e "git+https://github.com/Azure/azure-sdk-for-python#subdirectory=azure-sdk-tools&egg=azure-sdk-tools"
+     
+     mkdir dist
```

Comment:
If you are going to control the output just for publishing artifacts you can simply use `Build_ArtifactStagingDirectory` which is the default staging directory for output publishing. That is what I'm doing for the Java definition see https://github.com/Azure/azure-sdk-for-java/pull/2668/files#diff-ec4e3bfd6c71c48bef2e74f84f92cf88R26.

Reply:
Ok. I'll take a look and make adjustments.